content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -107,14 +158,27 @@ you can specify a time zone for a CronJob (if you don't enable that feature gate
 Kubernetes that does not have experimental time zone support, all CronJobs in your cluster have an unspecified
 timezone).
 
-When you have the feature enabled, you can set `spec.timeZone` to the name of a valid [time zone](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones). For example, setting
-`spec.timeZone: "Etc/UTC"` instructs Kubernetes to interpret the schedule relative to Coordinated Universal Time.
+When you have the feature enabled, you can set `.spec.timeZone` to the name of a valid [time zone](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones). For example, setting
+`.spec.timeZone: "Etc/UTC"` instructs Kubernetes to interpret the schedule relative to Coordinated Universal Time.
+
+{{< caution >}}
+Historically you may set the `.spec.schedule` field to a timezone like `CRON_TZ=UTC * * * * *` or
```

Comment:
This was never supported, so it should not be said as historically, but rather explicitly call it out as unsupported extension. Using is will give you warnings coming from the server whenever working with such a cronjob. Additionally, you can't use this unsupported with `.spec.timeZone`. 
I'd like to make this very explicit!

Reply:
Both `CRON_TZ` and just `TZ` do(see [create](https://github.com/kubernetes/kubernetes/blob/bc61d70055023607f5adca83a624a976a235ff7b/pkg/registry/batch/cronjob/strategy.go#L135) and [update](https://github.com/kubernetes/kubernetes/blob/bc61d70055023607f5adca83a624a976a235ff7b/pkg/registry/batch/cronjob/strategy.go#L171)), since that was not the official way, but rather it was an internal thing that started being supported by the library we use for parsing cron (after one of the upgrades). That's why it's important to make that distinction clear in the docs as well. I don't want to confuse users :cold_sweat: but rather suggest them to use supported paths.